lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1051,35 +1051,55 @@ sub update_backend {
 }
 
 sub insert_module {
-    my ($self, $tm) = @_;
-
+    my ($self, $tm, $skip_jobs_update) = @_;
+
+    # prepare query to insert job module
+    my $insert_sth = $self->{_insert_job_module_sth};
+    $insert_sth = $self->{_insert_job_module_sth} = $self->result_source->schema->storage->dbh->prepare(
+        <<'END_SQL'
+        INSERT INTO job_modules (
+            job_id, name, category, script, milestone, important, fatal, always_rollback, t_created, t_updated
+        ) VALUES(
+            ?,      ?,    ?,        ?,      ?,         ?,         ?,     ?,               now(),      now()
+        ) ON CONFLICT DO NOTHING
+END_SQL
```

Comment:
I consider https://github.com/os-autoinst/openQA/pull/2617/files#diff-85ae48e70a5c110c9e439c3a5ea28d5fR1121-R1124 more readable. Did you on purpose do it in this way here?

Reply:
I don't think there's a big difference in readability except that my version is better in the sense that you see which question mark belongs to which column.

Reply:
I find the indented version a lot easier to read